src/plugins/intel_gna/backend/pwl_border_values_counter.hpp

```diff
@@ -5,6 +5,7 @@
 #pragma once
 
 #include <cstdint>
+#include <limits>
```

Comment:
Maybe there is a sense to put this include into the appripriate file (`pwl_border_values_counter_identity.cpp`) instead of header?

Reply:
ok, moved in the current revision